Amethyst/Managers/WindowModifier.swift

```diff
@@ -294,6 +294,12 @@ extension ScreenFocuser {
             return
         }
 
+        // If the previous focus has been tracked, then focus the window that hat the focus before.
+        if let previouslyFocused = screenManager.lastFocusedWindow {
+            previouslyFocused.focus()
```

Comment:
This needs to be `am_focusWindow()` to get mouse tracking for mouse-follows-focus.